<?xml version="1.0" encoding="UTF-8"?>
<TestRun id="8eb3f83a-4b16-4d60-b034-ad27bf30faea" name="Javi@JAVI-PC 2010-09-19 13:18:33" runUser="Javi-PC\Javi" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <TestSettings name="Local" id="da34d0b4-0511-46eb-bb91-669de211e68e">
    <Description>These are default test settings for a local test run.</Description>
    <Deployment enabled="false" runDeploymentRoot="Javi_JAVI-PC 2010-09-19 13_18_33">
      <DeploymentItem filename="C:\Code\WcfHttpAuth\WcfHttpAuth\bin\Debug\WcfHttpAuth.dll" />
    </Deployment>
    <Execution>
      <TestTypeSpecific />
      <AgentRule name="Execution Agents">
      </AgentRule>
    </Execution>
  </TestSettings>
  <Times creation="2010-09-19T13:18:33.7359877+02:00" queuing="2010-09-19T13:18:35.0830647+02:00" start="2010-09-19T13:18:35.1980713+02:00" finish="2010-09-19T13:18:37.4191984+02:00" />
  <ResultSummary outcome="Failed">
    <Counters total="2" executed="2" passed="1" error="0" failed="1" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
  </ResultSummary>
  <TestDefinitions>
    <UnitTest name="Given_credentials_are_invalid_then_should_return_401" storage="c:\code\wcfhttpauth\wcfhttpauth.integrationtests\bin\debug\wcfhttpauth.integrationtests.dll" id="66708f93-7293-f476-715a-1f4ac9ebac9e">
      <Execution id="373804f9-c2e5-4232-9316-9c35673184b9" />
      <TestMethod codeBase="C:/Code/WcfHttpAuth/WcfHttpAuth.IntegrationTests/bin/Debug/WcfHttpAuth.IntegrationTests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="WcfHttpAuth.IntegrationTests.When_calling_a_basic_auth_service, WcfHttpAuth.IntegrationTests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="Given_credentials_are_invalid_then_should_return_401" />
    </UnitTest>
    <UnitTest name="Given_credentials_are_valid_then_should_return_200" storage="c:\code\wcfhttpauth\wcfhttpauth.integrationtests\bin\debug\wcfhttpauth.integrationtests.dll" id="d2c4f2ed-2807-a47b-bc9e-9a4b6401e12e">
      <Execution id="cd0ca490-625c-420d-84c4-77885d1cc2f8" />
      <TestMethod codeBase="C:/Code/WcfHttpAuth/WcfHttpAuth.IntegrationTests/bin/Debug/WcfHttpAuth.IntegrationTests.DLL" adapterTypeName="Microsoft.VisualStudio.TestTools.TestTypes.Unit.UnitTestAdapter, Microsoft.VisualStudio.QualityTools.Tips.UnitTest.Adapter, Version=10.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a" className="WcfHttpAuth.IntegrationTests.When_calling_a_basic_auth_service, WcfHttpAuth.IntegrationTests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null" name="Given_credentials_are_valid_then_should_return_200" />
    </UnitTest>
  </TestDefinitions>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <TestEntries>
    <TestEntry testId="66708f93-7293-f476-715a-1f4ac9ebac9e" executionId="373804f9-c2e5-4232-9316-9c35673184b9" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="d2c4f2ed-2807-a47b-bc9e-9a4b6401e12e" executionId="cd0ca490-625c-420d-84c4-77885d1cc2f8" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <Results>
    <UnitTestResult executionId="373804f9-c2e5-4232-9316-9c35673184b9" testId="66708f93-7293-f476-715a-1f4ac9ebac9e" testName="Given_credentials_are_invalid_then_should_return_401" computerName="JAVI-PC" duration="00:00:01.6498107" startTime="2010-09-19T13:18:35.2100720+02:00" endTime="2010-09-19T13:18:37.3381937+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="373804f9-c2e5-4232-9316-9c35673184b9">
      <Output>
        <ErrorInfo>
          <Message>Assert.AreEqual failed. Expected:&lt;ConnectFailure&gt;. Actual:&lt;ProtocolError&gt;. </Message>
          <StackTrace>   at WcfHttpAuth.IntegrationTests.When_calling_a_basic_auth_service.Given_credentials_are_invalid_then_should_return_401() in C:\Code\WcfHttpAuth\WcfHttpAuth.IntegrationTests\When_calling_a_basic_auth_service.cs:line 28
</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="cd0ca490-625c-420d-84c4-77885d1cc2f8" testId="d2c4f2ed-2807-a47b-bc9e-9a4b6401e12e" testName="Given_credentials_are_valid_then_should_return_200" computerName="JAVI-PC" duration="00:00:00.0246991" startTime="2010-09-19T13:18:37.3411939+02:00" endTime="2010-09-19T13:18:37.3711956+02:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Passed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="cd0ca490-625c-420d-84c4-77885d1cc2f8">
    </UnitTestResult>
  </Results>
</TestRun>